drag_and_drop_v2/public/css/drag_and_drop.css

```diff
@@ -83,8 +83,14 @@
     transform: translate(-50%, -50%); /* These blocks are to be centered on their absolute x,y position */
 }
 
+/* Focused option */
+.xblock--drag-and-drop .drag-container .item-bank .option:focus,
+.xblock--drag-and-drop .drag-container .item-bank .option:hover {
+    transform: scale(1.1);
```

Comment:
@bradenmacdonald That's a good idea. Implemented in [672aa8c](https://github.com/edx-solutions/xblock-drag-and-drop-v2/commit/https://github.com/open-craft/xblock-drag-and-drop-v2/commit/672aa8c0b72e80d7932e7160ce7666767b57b023).